tests/stability/conftest.py

```diff
@@ -1,16 +0,0 @@
-import pytest
-
-
-def pytest_collection_modifyitems(config, items):
-    # Ensure stability tests use spot instances by default
-    marker = pytest.mark.backend_options(
-        spot=True, spot_on_demand_fallback=True, multizone=True
-    )
-    for item in items:
-        # Add a module-level `spot=True` backend option marker if one doesn't already exist
-        module = item.parent
-        if not any(
-            m.kwargs.get("spot", False)
-            for m in module.iter_markers(name="backend_options")
-        ):
-            module.add_marker(marker)
```

Comment:
@jrbourbeau why did you enable spot=True, spot_on_demand_fallback=True, multizone=True for `stability` tests, but not for `benchmark` or `runtime`?

Reply:
there are some specific tests where I chose not to use spot because it seemed like the tests were more sensitive to cluster start time and there's the potential for some small impact from using spot + fallback (since we might get some spot, then need to make second request to get some on-demand, and I don't know if this is true but it's possible time to provision would be different on spot vs on-demand).

Reply:
Cluster start time is not included in the test runtime measure though... unless spot instances are frequently forcibly shut down in the middle of a test?